README.md

```diff
@@ -79,3 +80,6 @@ You can also use this file as starting point for your production docker-compose
 ### License
 
 Copyright © [GNU General Public License v3.0](./LICENSE.md)
+
+
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Freactioncommerce%2Freaction.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2Freactioncommerce%2Freaction?ref=badge_large)
```

Comment:
Not sure we need two badges in our readme.

Reply:
They're different, with a larger version down in the License section. Here's a render:

![screenshot](https://www.evernote.com/l/AAGgp5A1ah1IqrrQxiXhSrT1mY_SXJAdmZgB/image.png)